ci-operator/config/openshift/cluster-api-provider-azure/OWNERS

```diff
@@ -1,20 +1,21 @@
 # DO NOT EDIT; this file is auto-generated using tools/populate-owners.
-# from https://github.com/openshift/cluster-api-provider-azure/blob/35cc61a8e6081bccfbbd704c2e911d72809b8444/OWNERS
+# from https://github.com/openshift/cluster-api-provider-azure/blob/master/OWNERS when master is at 3ac8354bd5c18a976b72998e9ff196aeac9836ac
 # See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
 
 approvers:
+- bison
 - enxebre
 - frobware
 - ingvagabund
-- bison
+options: {}
```

Comment:
Ugly.

Reply:
Maybe `omitempty` it upstream?

Reply:
Would it work on struct (not pointer)?
https://www.sohamkamani.com/blog/golang/2018-07-19-golang-omitempty/

Reply:
Unfortunately no :(